kbase-extension/static/kbase/js/widgets/appWidgets2/view/sequenceView.js

```diff
@@ -200,75 +156,70 @@ define([
               Set focus on the new input control
         */
 
-        function addNewControl(initialValue) {
+        function addNewControl(initialValue, initialDisplayValue) {
```

Comment:
Here's the main change to the sequence view - pass along the initial display value to the various sub widgets.